### PR TITLE
feat: add SEO tags (#745)

### PR DESCRIPTION
**Pull request by EtayZaslavsky**
# Description
Added SEO tags to index.html. added an image version of the svg logo for social networks' SEO.
Please review @NoamGaash :)

### CODE REVIEW

**Review by R894 — APPROVED** [review]
Looks good to me!

Below are some suggestions  I think might help us

If we want to take it a step further its also possible to add schema.org tags
```html
  <script type="application/ld+json">
  {
    "@context": "http://schema.org",
    "@type": "Website",
    "name": "דאטאבוס",
    "url": "https://open-bus-map-search.hasadna.org.il/",
    "description": "אתר דאטאבוס מציג נתונים על איכות קווי התחבורה הציבורית בארץ, מתוך מקורות מידע ממשלתיים.",
    "sameAs": [
      "https://www.facebook.com/hasadna.org.il/",
      "https://x.com/hasadna"
    ]
  }
  </script>
  ```

After running the site through an SEO checker it also suggested using header tags ( ```<h1> - <h6>``` ), maybe we an add that somewhere also

File: index.html [hunk 1]
```diff
@@ -1,53 +1,80 @@
...
+  <meta name="theme-color" content="#000000" />
+  <meta name="Open-Bus" content="Open Bus web app" />
+  <link rel="apple-touch-icon" href="logo192.png" />
+  <link rel="manifest" href="manifest.json" />
+
+  <!-- Primary Meta Tags -->
+  <title>דאטאבוס</title>
+  <meta name="title" content='דאטאבוס - נתוני ביצוע לתחב"צ הישראלי' />
+  <meta name="description"
+    content="אתר דאטאבוס מציג נתונים על איכות קווי התחבורה הציבורית בארץ, מתוך מקורות מידע ממשלתיים, במטרה להביא תמונת מצב על תפקוד התחבורה הציבורית בארץ." />
```
Comment (NoamGaash) [inline index.html:17]: :rocket:

File: public/social-image.png
Comment (NoamGaash) [inline public/social-image.png:1]: That's a weird version of the logo, but why not? I like it.

**Review by NoamGaash — APPROVED** [review]
Thank you! It looks great, and solve 99% of the problem. Looking forward to see it merged :partying_face: 
Let's leave the issue open, as we can still use some more contributions regarding canonical links and a sitemap

**Comment by EtayZaslavsky** [comment]
Great!
@NoamGaash - Will be onto the other tasks in few days or so.
@R894 - I will definitely add schema.org tags.

**Comment by NoamGaash** [comment]
Thank you! :partying_face: 


@all-contributors please add @EtayZaslavsky for code

**Pull request merged**